.claude/skills/add-feature-module/SKILL.md

```
@@ -0,0 +1,185 @@
+### 3. Registration
+1. `settings.gradle.kts`: Add the new module to the feature modules block in alphabetical order:
+```kotlin
+include(":feature:{module}")
+```
+
+2. Sync & Verify:
+```bash
+./gradlew :feature:{module}:compileDebugKotlin
+```
+## Naming & Architecture Rules
+| Item                 | Convention                                 | Example(module=`person_detail`)                |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)